### PR TITLE
Feature: Support unit test in go.2

### LINKED ISSUE #117: Feature: Support unit test in go.
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [OpenIMSDK Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) and got positive feedback

### Is this feature request related to a problem?

❎ No

### Problem Description

Support go test and get code cover

- https://github.com/agiledragon/gomonkey
- https://blog.csdn.net/xidianhuihui/article/details/131128806

### Solution Description

Add code coverage tool in /server

### Benefits

Enhance system.

### Potential Drawbacks

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by a67793581**
🔍 What type of PR is this?
Enhance system.

Add one of the following kinds:
/kind feature

Optionally add one or more of the following kinds if applicable:
/kind failing-test

👀 What this PR does / why we need it:
 My pull request adheres to the code style of this project
 My code requires changes to the documentation
 I have updated the documentation as required
 All the tests have passed
🅰 Which issue(s) this PR fixes:
Fix #117 

📝 Special notes for your reviewer:
🎯 Describe how to verify it
https://github.com/a67793581/OpenKF/tree/feat/test/server/test

📑 Additional documentation e.g., RFC, notion, Google docs, usage docs, etc.:
https://github.com/a67793581/OpenKF/tree/feat/test/server/test

### CODE REVIEW

File: server/test/log_test.go [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+	"github.com/sirupsen/logrus"
+	"testing"
+)
+
+func TestInitLogger(t *testing.T) {
+	tests := []struct {
+		name string
+	}{
+		{name: "测试-InitLogger"},
+	}
```
Comment (charles-chenzz) [inline server/test/log_test.go:15]: please use English for all the naming.

**Comment by charles-chenzz** [comment]
please fix CI failure and address comment
ref https://github.com/openimsdk/openkf/pull/178/files#r1320994812
the commit message need to be update as well. Thanks!

File: server/test/log_test.go (hunk 1 above)
Comment (a67793581) [reply server/test/log_test.go:15]: ok

**Review by cubxxw — APPROVED** [review]
lgtm

**Pull request merged**